CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -11,150 +11,134 @@ import UIKit
-            string: "Pronouns",
-            attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray]
-        )
-        pronounsTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
-        pronounsTextField.leftViewMode = .always
-        pronounsTextField.layer.cornerRadius = fieldsCornerRadius
-        view.addSubview(pronounsTextField)
+        // Renders the valid graduation years based on current year.
+        let currentYear = Calendar.current.component(.year, from: Date())
+        let gradYear = currentYear + 4 // Allow only current undergrads and fifth years
+        classSearchFields = (currentYear...gradYear).map { "\($0)" }
+
+        classDropdownView = OnboardingSelectDropdownView(delegate: self, placeholder: "Class of...", tableData: classSearchFields, textTemplate: "Class of")
+        classDropdownView.tag = 0 // Set tag to keep track of field selection status.
+        view.addSubview(classDropdownView)
+
+        majorDropdownView = OnboardingSearchDropdownView(delegate: self, placeholder: "Major", tableData: majorSearchFields)
+        majorDropdownView.tag = 1 // Set tag to keep track of field selection status.
+        view.addSubview(majorDropdownView)
+
+        hometownDropdownView = OnboardingSearchDropdownView(delegate: self, placeholder: "Hometown", tableData: hometownSearchFields)
+        hometownDropdownView.tag = 2 // Set tag to keep track of field selection status.
+        view.addSubview(hometownDropdownView)
+
+        pronounsDropdownView = OnboardingSelectDropdownView(delegate: self, placeholder: "Pronouns", tableData: pronounSearchFields, textTemplate: "")
+        pronounsDropdownView.tag = 3 // Set tag to keep track of field selection status.
+        view.addSubview(pronounsDropdownView)
 
         nextButton.setTitle("Next", for: .normal)
-        nextButton.setTitleColor(.textBlack, for: .normal)
-        nextButton.backgroundColor = .backgroundLightGray
+        nextButton.setTitleColor(.white, for: .normal)
+        nextButton.titleLabel?.font = ._20CircularStdBold
+        nextButton.backgroundColor = .inactiveGreen
+        nextButton.isEnabled = false
         nextButton.layer.cornerRadius = 26
         nextButton.addTarget(self, action: #selector(nextButtonPressed), for: .touchUpInside)
         view.addSubview(nextButton)
 
         setUpConstraints()
     }
 
-    @objc func nextButtonPressed() {
-        delegate?.nextPage(index: 1)
+    @objc private func nextButtonPressed() {
+        delegate.nextPage(index: 1)
     }
 
     private func setUpConstraints() {
+        let fieldTopPadding: CGFloat = LayoutHelper.shared.setCustomVerticalPadding(size: 60)
         let helloLabelHeight: CGFloat = 30
-        let helloLabelSpacing: CGFloat = 100
-        let nextBottomPadding: CGFloat = 90
+        let helloLabelPadding: CGFloat = LayoutHelper.shared.setCustomVerticalPadding(size: 100)
```

Comment:
should the function be renamed to `getCustomVerticalPadding` instead of `set` since its returning a value